### PR TITLE
Allow setting Certificate secretTemplate

### PR DESCRIPTION
**Pull request by pharaujo**
Some environments have admission webhooks that might mandate specific labels or annotations. This change allows chart users to set labels and/or annotations in the Secret created by the Certificate resources.

### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] README is updated with new configuration values *(if applicable)*

### CODE REVIEW

File: keda/README.md [hunk 1]
```diff
@@ -215,6 +215,7 @@ their default values.
 | `certificates.certManager.enabled`                         | Enables Cert-manager for certificate management | `false` |
 | `certificates.certManager.generateCA`                      | Generates a self-signed CA with Cert-manager | `true` |
 | `certificates.certManager.caSecretName`                    | Secret name where the CA is stored (generatedby cert-manager or user given) | `kedaorg-ca` |
+| `certificates.certManager.secretTemplate`                  | Labels or annotations to add to the secret generated by cert-manager | `{}` |
```
Comment (JorTurFer) [inline keda/README.md:218]: I'd add a link to the official docs to be clearer about the usage, WDYT?
```suggestion
| `certificates.certManager.secretTemplate`                  | [Labels or annotations to add to the secret generated](https://cert-manager.io/docs/usage/certificate/#creating-certificate-resources) by cert-manager | `{}` |
```

File: keda/values.yaml [hunk 2]
```diff
@@ -448,6 +448,14 @@ certificates:
     # has to be annotated with 'cert-manager.io/allow-direct-injection: "true"'
     generateCA: true
     caSecretName: "kedaorg-ca"
+    # Add labels/annotations to secrets created by Certificate resources
+    secretTemplate:
+      {}
```
Comment (JorTurFer) [inline keda/values.yaml:453]: Other cases are inline
```suggestion
    secretTemplate: {}
```

**Review by JorTurFer — COMMENTED** [review]
Looking good, some nits inline

**Review by JorTurFer — APPROVED** [review]
LGTM!
Thanks for the improvement! ❤️

**Comment by pharaujo** [comment]
Thanks for such a quick review and merge! Out of curiosity, what's the chart release cycle like?

**Pull request merged**